test-utils/src/main/scala/com/mesosphere/utils/mesos/MesosTest.scala

```diff
@@ -328,6 +328,7 @@ case class MesosCluster(
       "MESOS_SYSTEMD_ENABLE_SUPPORT" -> "false",
       "MESOS_SWITCH_USER" -> "false"
     ) ++
+      config.restrictedToRoles.map("MESOS_ROLES" -> _).to[Seq] ++
```

Comment:
So, you probably want to remove `"MESOS_ROLES" -> "public,test"` line 325 above?

Reply:
Man, I was wondering how our integration tests worked 🙈 Thanks.